softlayer/stemcell/softlayer_finder_test.go

```diff
@@ -49,4 +49,26 @@ var _ = Describe("SoftLayerFinder", func() {
 			})
 		})
 	})
+
+	Describe("FindById", func() {
+		Context("valid stemcell ID pointing to a SL virtual disk image", func() {
+			It("returns stemcell and found as true if stemcell ", func() {
+				stemcell, found, err := finder.FindById(4868344)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(found).To(BeTrue())
+
+				expectedStemcell := NewSoftLayerStemcell(4868344, "8c7a8358-d9a9-4e4d-9345-6f637e10ccb7", VirtualDiskImageKind, softLayerClient, logger)
+				Expect(stemcell).To(Equal(expectedStemcell))
+			})
+		})
+
+		Context("valid stemcell ID pointing to a SL virtual disk image", func() {
+			It("returns found as false if stemcell directory does not exist", func() {
```

Comment:
can we fix the `It` string to be better. Seems that "stemcell directory does not exist" is not correct description...